src/main/java/com/google/sps/data/NSWGovAQDataPoint.java

```diff
@@ -17,6 +17,14 @@ public NSWGovAQDataPoint() {
     this.aqi = -1;
   }
 
+  public int getId() {
```

Comment:
```suggestion
  public int getSiteId() {
```

Reply:
@Techget  thanks for the Suggestion! Though I think if i make this a PDS, then there is no need for a getter anymore